src/CommandHandler/Wishlist/AddSelectedProductsToCartHandler.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/*
+ * This file was created by developers working at BitBag
+ * Do you need more information about us and what we do? Visit our https://bitbag.io website!
+ * We are hiring developers from all over the world. Join us and start your new, exciting adventure and become part of us: https://bitbag.io/career
+*/
+
+declare(strict_types=1);
+
+namespace BitBag\SyliusWishlistPlugin\CommandHandler\Wishlist;
+
+use BitBag\SyliusWishlistPlugin\Command\Wishlist\AddSelectedProductsToCart;
+use BitBag\SyliusWishlistPlugin\Command\Wishlist\AddWishlistProduct;
+use Sylius\Component\Core\Repository\OrderRepositoryInterface;
+use Sylius\Component\Order\Modifier\OrderItemQuantityModifierInterface;
+use Sylius\Component\Order\Modifier\OrderModifierInterface;
+use Symfony\Component\HttpFoundation\Session\Flash\FlashBagInterface;
+use Symfony\Component\Messenger\Handler\MessageHandlerInterface;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+final class AddSelectedProductsToCartHandler implements MessageHandlerInterface
+{
+    private FlashBagInterface $flashBag;
+
+    private TranslatorInterface $translator;
+
+    private OrderItemQuantityModifierInterface $itemQuantityModifier;
+
+    private OrderModifierInterface $orderModifier;
+
+    private OrderRepositoryInterface $orderRepository;
+
+    public function __construct(
+        FlashBagInterface $flashBag,
+        TranslatorInterface $translator,
+        OrderItemQuantityModifierInterface $itemQuantityModifier,
+        OrderModifierInterface $orderModifier,
+        OrderRepositoryInterface $orderRepository
+    ) {
+        $this->flashBag = $flashBag;
+        $this->translator = $translator;
+        $this->itemQuantityModifier = $itemQuantityModifier;
+        $this->orderModifier = $orderModifier;
+        $this->orderRepository = $orderRepository;
+    }
+
+    public function __invoke(AddSelectedProductsToCart $addSelectedProductsToCart): void
+    {
+        /** @var AddWishlistProduct $wishlistProduct */
+        foreach ($addSelectedProductsToCart->getWishlistProducts() as $wishlistProduct) {
+            if ($wishlistProduct->isSelected()) {
```

Comment:
This could be reversed: negate and use continue; will improve readability and decrease indentation.